operators/test/e2e/smoke_test.go

```diff
@@ -0,0 +1,36 @@
+package e2e
+
+import (
+	"bufio"
+	"os"
+	"testing"
+
+	"github.com/elastic/cloud-on-k8s/operators/test/e2e/apm"
+	"github.com/elastic/cloud-on-k8s/operators/test/e2e/helpers"
+	"github.com/elastic/cloud-on-k8s/operators/test/e2e/stack"
+	"k8s.io/apimachinery/pkg/util/yaml"
+)
+
+func TestSmoke(t *testing.T) {
+	var sampleStack stack.Builder
+	var sampleApm apm.Builder
+
+	yamlFile, err := os.Open(sampleEsApmFile)
+	helpers.ExitOnErr(err)
+
+	decoder := yaml.NewYAMLToJSONDecoder(bufio.NewReader(yamlFile))
+	helpers.ExitOnErr(decoder.Decode(&sampleStack.Elasticsearch))
+	helpers.ExitOnErr(decoder.Decode(&sampleApm.ApmServer))
+	helpers.ExitOnErr(decoder.Decode(&sampleStack.Kibana))
+
+	namespacedSampleStack := sampleStack.WithNamespace(helpers.DefaultNamespace)
+	namespacedSampleApm := sampleApm.WithNamespace(helpers.DefaultNamespace)
+
+	k := helpers.NewK8sClientOrFatal()
+	helpers.TestStepList{}.
+		WithSteps(stack.InitTestSteps(namespacedSampleStack, k)...).
+		WithSteps(apm.InitTestSteps(namespacedSampleApm, k)...).
+		WithSteps(stack.CreationTestSteps(namespacedSampleStack, k)...).
+		WithSteps(apm.CreationTestSteps(namespacedSampleApm, namespacedSampleStack.Elasticsearch, k)...).
```

Comment:
To be sure: we don't use the `DeletionTestSteps` to gain a few seconds in the total running time?

Reply:
We don't use `DeletionTestSteps`. I can add them if you feel it will make tests better

Reply:
I checked and with `DeletionTestSteps` test runs ~ 1 min more (205 sec vs 157 sec). Main question will it be helpful? That do you think @thbkrkr ?